mypy_django_plugin/main.py

```diff
@@ -79,18 +80,54 @@ def exit(error_type: int) -> NoReturn:
         }
         handler.error("'django_settings_module' is not set: " + messages[error_type])
 
-    parser = configparser.ConfigParser()
-    try:
-        with open(cast(str, config_file_path)) as handle:
-            parser.read_file(handle, source=config_file_path)
-    except (IsADirectoryError, OSError):
-        exit(1)
-
-    section = "mypy.plugins.django-stubs"
-    if not parser.has_section(section):
-        exit(2)
-    settings = parser.get(section, "django_settings_module", fallback=None) or exit(3)
-    return cast(str, settings).strip("'\"")
+    def exit_toml(error_type: int) -> NoReturn:
+        from mypy.main import CapturableArgumentParser
+
+        usage = """(config)
+        ...
+        [tool.django-stubs]
+        django_settings_module = str (required)
+        ...
+        """.replace(
```

Comment:
There's https://docs.python.org/3/library/textwrap.html#textwrap.dedent 
Maybe it is better?

Reply:
I added that change in an isolated commit just in case you don't like so it's easier to revert 😆 

https://github.com/typeddjango/django-stubs/pull/639/commits/a503ea1de5f41bd5bd6f57449551700fe6947513

The only problem that I found with `textwrap.dedent` is that the first line must also be indented otherwise it doesn't work. So this won't work anymore:

```
    usage = """(config)
    ....
    """
```

and I had to change it to be like this:

```
    usage = """
    (config)
    ....
    """
```

And that actually changes how the django-stubs displays those errors to the user - hence the updated tests.

Reply:
Hmm I just realized that I could `.lstrip()` the string no changes in the tests are necessary:

```
handler = CapturableArgumentParser(prog="(django-stubs) mypy", usage=textwrap.dedent(usage).lstrip())
```

What do you think?